lib/src/chip/Chip.stories.tsx

```diff
@@ -23,20 +46,29 @@ export const Chromatic = () => (
       <DxcChip label="Default Chip" />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with prefix" theme="light" level={4} />
-      <DxcChip label="Chip with prefix" prefixIcon={iconSVG} />
+      <Title title="Chip with prefix SVG (small icon)" theme="light" level={4} />
+      <DxcChip label="Chip with prefix" prefixIcon={smallIconSVG} />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with suffix" theme="light" level={4} />
+      <Title title="Chip with suffix SVG (large icon)" theme="light" level={4} />
       <DxcChip label="Chip with suffix" suffixIcon={iconSVG} />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with prefix and suffix" theme="light" level={4} />
-      <DxcChip label="Chip with prefix and suffix" prefixIcon={iconSVG} suffixIcon={iconSVG} />
+      <Title title="Chip with prefix (SVG) and suffix (URL)" theme="light" level={4} />
+      <DxcChip
+        label="Chip with prefix and suffix"
+        prefixIcon={iconSVG}
+        suffixIcon="https://upload.wikimedia.org/wikipedia/commons/e/e0/Check_green_icon.svg"
+      />
     </ExampleContainer>
     <ExampleContainer>
       <Title title="Disabled chip" theme="light" level={4} />
-      <DxcChip label="Disabled" disabled prefixIcon={iconSVG} suffixIcon={iconSVG} />
+      <DxcChip
+        label="Disabled"
+        disabled
+        prefixIcon={iconSVG}
+        suffixIcon="https://upload.wikimedia.org/wikipedia/commons/e/e0/Check_green_icon.svg"
```

Comment:
Yes... But that was a premeditated decision. Look for example the action of a Text Input when it receives an image:

![image](https://user-images.githubusercontent.com/44321109/214333228-1f84a299-2828-4cd1-8b90-8543f327320c.png)

This is because color styling cannot be properly applied to images and we don't usually style with `opacity` instead we use opacity in the colors. As you can read in the docs:

![image](https://user-images.githubusercontent.com/44321109/214334535-e0eba9c2-f24e-4b9b-82a4-5dae8a4330fd.png)